src/modules.js

```diff
@@ -516,6 +516,7 @@ var PassManager = {
       ATINITS: ATINITS.join('\n'),
       ATMAINS: ATMAINS.join('\n'),
       ATEXITS: ATEXITS.join('\n'),
+      EXTRA_WASM_IMPORTS: EXTRA_WASM_IMPORTS,
```

Comment:
This allows for the setting to be mutated by the js compiler?

Seems a little hacky, but maybe fine.

Reply:
I think it's worth it to do things like define the autodebug functions and ask that they be imported, all in 1 file.